src/charm.py

```diff
@@ -375,12 +375,7 @@ def _on_install(self, _event: InstallEvent) -> None:
                     ssh_debug_connections=state.ssh_debug_connections,
                 )
                 logger.info("OpenStack instance: %s", instance)
-            # Test out openstack integration and then go
-            # into BlockedStatus as it is not supported yet
-            self.unit.status = BlockedStatus(
-                "OpenStack integration is not supported yet. "
-                "Please remove the openstack-clouds-yaml config."
-            )
+            self._block_on_openstack_config(state)
```

Comment:
The order of the fct is inconsistent with our contribution guide https://github.com/canonical/is-charms-contributing-guide/blob/main/CONTRIBUTING.md#function-and-method-Ordering (_block_on_openstack_config should be listed before this fct), but it doesn't seem to be followed in a lot of other PRs in our repos, so perhaps this rule should be revised in the contribution guide, or stated as a recommendation, otherwise I'll have to mention this nitpick in a lot of PRs. cc @jdkandersson Feel free to resolve this conversation.

Reply:
Updated, thank you!